src/main/java/omero/gateway/facility/TablesFacility.java

```diff
@@ -123,6 +149,8 @@ public TableData addTable(SecurityContext ctx, DataObject target,
             anno.setFile(file);
             FileAnnotationData annotation = new FileAnnotationData(anno);
             annotation.setDescription(name);
+            if (ns != null && ns.trim().length() > 0)
```

Comment:
Maybe ``StringUtils.isBlank()`` could be used here. I think apache package is already in the dependencies tree

Reply:
sorry ignore, since we are trying not to use the dependency